### ISSUE TITLE
Adobe Acrobat Reader DC: print dialog and preferences dialog unreadable

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/scripts#849]
![deepinscreenshot_select-area_20190131212045](https://user-images.githubusercontent.com/3283676/52066279-3eca0c00-259e-11e9-98f2-3902e1676f54.png)
![deepinscreenshot_select-area_20190131212029](https://user-images.githubusercontent.com/3283676/52066280-3eca0c00-259e-11e9-8c55-61682bf06938.png)

see https://github.com/flathub/org.phoenicis.playonlinux/issues/3

**Comment by aiamuzz** [comment]
> This can be solved by using Windows XP instead of Windows 7.

how do i choose Windows xp ... the application is installing all by itself ... i couldn't find an option to select/choose the version of Windows ... please guide me on how i can change it from Windows 7 to XP ... do i have to reinstall or i can edit the current install itself ?

**Comment by aiamuzz** [comment]
I am trying to use the online installer ... will report back on how it goes ...

**Comment by aiamuzz** [comment]
ok ... i tried installing it using the online installer ...

donwload url : 
https://ardownload2.adobe.com/pub/adobe/reader/win/AcrobatDC/1901020069/AcroRdrDC1901020069_en_US.exe

using this information ...


Version: | DC
License: | Free to use
URL: | http://www.adobe.com/products/...
Votes: | 0
Latest Rating: | Gold
Latest Wine Version Tested: | 3.21-staging

on amd64 and Windows 7 ... turns out the installation fails at the last minute ...

![deepinscreenshot_msiexec exe_20190201010204](https://user-images.githubusercontent.com/3283676/52080255-56fd5380-25bd-11e9-96d5-cc56e9e02f8e.png)

**Comment by ImperatorS79** [comment]
@aiamuzz That is certainly because abobe reader needs mspatcha verbs.

**Comment by aiamuzz** [comment]
> @aiamuzz That is certainly because abobe reader needs mspatcha verb.

@ImperatorS79 ... how do i install this mspatcha verb ?

Is this the reason i am unable to install using Local and Online isntallation methods ?

**Comment by ImperatorS79** [comment]
Yes, there is an issue for this. Currently local and online installer cannot install verbs. However, you can install verbs in an existing container from the container tab.

**Comment by aiamuzz** [comment]
> Yes, there is an issue for this. Currently local and online installer cannot install verbs. However, you can install verbs in an existing container from the container tab.

Could you provide me the steps to install the verbs in an existing container ?

but do that the installation has to complete only then a container will be created isn't it ?

**Comment by ImperatorS79** [comment]
Yes, there is also an issue for manual container creation. However, container are not deleted if an installation fails, so you should have one.

Go to to the container tab click on the container. In the right pane, click on "Verbs". Then scroll until you find mspatcha.

**Comment by aiamuzz** [comment]
> Yes, there is also an issue for manual container creation. However, containers are not deleted if an installation fails, so you should have one.

sorry but i didn't find it ... I'll give it a try once again ...

> Go to to the container tab and click on the container. In the right pane, click on "Verbs". Then scroll until you find mspatcha.

thanks ... i just figured that out ... :)

**Comment by aiamuzz** [comment]
> Yes, there is also an issue for manual container creation. However, containers are not deleted if an installation fails, so you should have one.
> 
> Go to to the container tab and click on the container. In the right pane, click on "Verbs". Then scroll until you find mspatcha.

@ImperatorS79 ...

ok ... i chose Local Installer ... the container was created ... i also tried installing the verb as you suggested ... it seemed it was installing and then it closed by itself ... as i could not see any confirmation that the verb was installed successfully ... Any way i can check if the verb is installed ?

Now how do i attempt a reinstall of Adobe Reader into this already created container ?

**Comment by ImperatorS79** [comment]
This is a quite small verb so it should be quick.
If you try to reinstall adobe reader with online installer with the same container name I would assume it will use it.

**Comment by aiamuzz** [comment]
:disappointed: ... though the new attempts to provide a good and simple frontend to Wine are being made ... but sadly getting windows apps on linux is as difficult and time consuming as it was earlier ... only a guy with sufficient knowledge of linux and a lot of patience may succeed more often but a new linux user who has migrated from Windows ... things don't seem to be simple and easy !!!

I have spent 2 days and counting to get Adobe Reader to work without any luck !!!

I guess simple and easy is still far for a new and average linux user ...

**Comment by ImperatorS79** [comment]
Well, it is still an alpha. The issue you pointed out has a fix though.

**Comment by qparis** [comment]
Yeah it is alpha stage. Things will be improved

**Comment by aiamuzz** [comment]
oh ... ok ... i see it ... its just 2 releases old and as you said in alpha stage ... 

hmmmmm ... the fact that a flatpak version was created i assumed it must be a mature application ... that's how i got introduced to it to begin with ...

**Comment by qparis** [comment]
I am the original founder of the project yep (It was 13 years ago though)

**Comment by aiamuzz** [comment]
> I am the original founder of the project yep (It was 13 years ago though)

you mean original founder of playonlinux ? ... has it been around since 13 years ? 
wow that's good man ...

i have been a linux user for say 8-10 years ... its surprising i came across it only a year ago !!!

**Comment by qparis** [comment]
Yep.. To complete the story I was 14 when it was founded and I had hardly any software architecture skills. When the year passed, the project accumulated an important technical debt. 

We wanted to keep it alive and try to recreate it from sane principles. However, the time I have to spend on it is less important than the one I had when I was in school.

So we try with the new awesome team to do our best to give you something innovative as soon as we can !

**Comment by aiamuzz** [comment]
> Yep.. To complete the story I was 14 when it was founded and I had hardly any software architecture skills. When the year passed, the project accumulated an important technical debt.
> 
> We wanted to keep it alive and try to recreate it from sane principles. However, the time I have to spend on it is less important than the one I had when I was in school.
> 
> So we try with the new awesome team to do our best to give you something innovative as soon as we can !

wow 14 years ... damn ! ... wish you all the best on your efforts 

PS : I have emailed you on your domain email ID ... If you have time I would like to know what you feel about it ?
